classes/class-wc-connect-api-client.php

```diff
@@ -193,6 +193,7 @@ protected function request( $method, $path, $body = array() ) {
 				'last_heartbeat' => get_option( 'wc_connect_last_heartbeat', 0 ),
 				'last_rate_request' => get_option( 'wc_connect_last_rate_request', 0 ),
 				'active_services' => $this->wc_connect_loader->get_active_services(),
+				'enable_stats' => ! Jetpack::is_staging_site(),
```

Comment:
I'd like to suggest that this be `disable_stats` instead. It makes more sense semantically to have this be a flag to turn it off and have it be on by default.

This will simplify the server PR a little bit too, checking for a truthy value is easier to read then checking for `false` explicitly.

Also I think this value should get a filter on it; though it's optional